projectq/backends/_circuits/_drawer.py

```diff
@@ -244,7 +246,14 @@ def _print_cmd(self, cmd):
         for l in all_lines:
             self._qubit_lines[l].append(item)
 
-    def get_latex(self):
+        """
+        The commands are stored on a per qubit basis in a very strange manner
+        In order to draw according to the order how they were added here
+        I am storing the first qubit/line which the commands affect
+        """
+        self._paler_command_order.append(all_lines[0])
+
+    def get_latex(self, ordered = False, sequential_gates = False):
```

Comment:
Please document the new argument in the function docstring

Reply:
new PR addresses this comment